TestingHelpers/StringExtensions.cs

```diff
@@ -55,5 +55,33 @@ public static string Replace(this string source, string oldValue, string newValu
 
             return result.ToString();
         }
+
+        [Pure]
+        public static string TrimSlashes(this string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return path;
+            }
+
+            var trimmed = path.TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
+
+            if (MockUnixSupport.IsUnixPlatform()
+                && (path[0] == Path.DirectorySeparatorChar || path[0] == Path.AltDirectorySeparatorChar)
+                && trimmed == "")
```

Comment:
string.Empty?

Reply:
Eh, I prefer `""` to `string.Empty`. 2 chars vs 12.